package.json

```diff
@@ -43,5 +43,8 @@
     "url-search-params-polyfill": "1.3.0",
     "velocity-animate": "^1.5.0",
     "whatwg-fetch": "^3.0.0"
+  },
+  "resolutions": {
+    "graceful-fs": "^4.2.9"
```

Comment:
You need this with node 10?

Reply:
@kilemensi We need this with node v14.15.0.